packages/manager/tools/component-rollup-config/README.md

```diff
@@ -0,0 +1,206 @@
+# Component rollup config
+
+> Extensible rollup configuration to build OVH components.
+
+[![Downloads](https://badgen.net/npm/dt/@ovh-ux/component-rollup-config)](https://npmjs.com/package/@ovh-ux/component-rollup-config) [![Dependencies](https://badgen.net/david/dep/ovh-ux/component-rollup-config)](https://npmjs.com/package/@ovh-ux/component-rollup-config?activeTab=dependencies) [![Dev Dependencies](https://badgen.net/david/dev/ovh-ux/component-rollup-config)](https://npmjs.com/package/@ovh-ux/component-rollup-config?activeTab=dependencies) [![Gitter](https://badgen.net/badge/gitter/ovh-ux/blue?icon=gitter)](https://gitter.im/ovh/ux)
```

Comment:
Both `Dependencies` and `Dev Dependencies` must be updated (due to the new path inside the monorepo).

**Example for `Dependencies`:**
`https://badgen.net/david/dep/ovh-ux/component-rollup-config` > `https://badgen.net/david/dep/ovh-ux/manager/packages/manager/tools/component-rollup-config`